qemu/tests/cfg/blockdev_stream_to_invalid_node.cfg

```diff
@@ -33,3 +33,13 @@
     virtio_blk:
         qmp_error_before_6_1 = "Conflicts with use by /machine/peripheral/${device_tag}/virtio-backend as 'root', which uses 'write' on %s"
         qmp_error_since_6_1 = "permissions 'write' are both required by block device '/machine/peripheral/${device_tag}/virtio-backend' (uses node '%s' as 'root' child)"
+    iscsi_direct:
+        lun_image1 = 0
```

Comment:
remove this line in case you want to assign a different lun other than 0

Reply:
Have updated, thanks zhenchao.